resources/resources.go

```diff
@@ -383,6 +385,10 @@ func (obj *BaseRes) Close() error {
 	close(obj.stopped)
 	obj.waitGroup.Done()
 
+	if err := obj.Prometheus().RemoveManagedResource(fmt.Sprintf("%v[%v]", obj.Kind(), obj.GetName()), obj.kind); err != nil {
+		return errwrap.Wrapf(err, "Could not increase Prometheus counter!")
```

Comment:
error message should not start with a capital letter or end with punctuation in order to pass golint.
some comment for other messages.